doc/source/methods/tabulardrift.ipynb

```diff
@@ -35,7 +35,7 @@
     "\n",
     "* `p_val`: p-value used for significance of the K-S and Chi-Squared test across all features. If the FDR correction method is used, this corresponds to the acceptable q-value.\n",
     "\n",
-    "* `categories_per_feature`: Dictionary with as keys the column indices of the categorical features and optionally as values the number of possible categorical values for that feature. E.g.: *{0: 5, 1: 9, 2: 7}* or *{0: None, 1: None, 2: None}*. If this argument is not specified, no features are assumed to be categorical.\n",
+    "* `categories_per_feature`: Dictionary with as keys the column indices of the categorical features and optionally as values the number of possible categorical values for that feature or a list with the possible values. E.g.: *{0: 4, 1: 2, 2: 3}* or *{0: None, 1: None, 2: None}* or *{0: [0, 1, 2, 3], 1: [0, 1], 2: [0, 1, 2]}*. If this argument is not specified, no features are assumed to be categorical.\n",
```

Comment:
I think we need more clear documentation on exactly which format is appropriate for which use case, e.g. `None` stuff for just identifying categorical columns, the first one for identifying number of of categories (all assumed to be in `[0, ..., n-1]` ?) and the last one for identifying categories by value explicitly (i.e. not assumed to be in `[0, ..., n-1]`).